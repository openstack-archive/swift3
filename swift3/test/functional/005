#!/bin/bash

# DELETE Bucket

. ./common

S3USER=tester

_s3_put /bucket

_s3_delete /bucket
