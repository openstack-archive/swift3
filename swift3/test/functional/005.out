QA output created by 005
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X DELETE http://SWIFT_HOST/bucket... 204
